kind/actions/cluster_actions.go

```diff
@@ -379,3 +346,8 @@ func KubeadmReset(clusterName, nodeName string) error {
 
 	return nil
 }
+
+// ProviderID formats the provider id needed to set on the node
+func ProviderID(name string) string {
```

Comment:
Put this logic in ONE place instead of having it split in two places with the wrong provider ID